ethereum/api/src/main/resources/schema.graphqls

```diff
@@ -199,6 +199,14 @@ type Block {
   withdrawals are unavailable for this block, this field will be null.
   """
   withdrawals: [Withdrawal!]
+
+  """BlobGasUsed is the total amount of gas used by the transactions."""
```

Comment:
```suggestion
  """BlobGasUsed is the total amount of blob gas used by the transactions."""
```

Reply:
This is a cut and paste from the execution-apis repo, it should be fixed there first - https://github.com/ethereum/execution-apis/pull/468/files#diff-a44507d2ad2482bb3ed863208b9e1e0393069edfdbd686593bac598901f36770R804